gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-all.zip
+distributionSha256Sum=13bf8d3cf8eeeb5770d19741a59bde9bd966dd78d17f1bbad787a05ef19d1c2d
```

Comment:
Including the checksum validation is a good change,  but it does result in a somewhat confusing message from IntelliJ if you have the android plugin enabled (which is the default). You can fairly easily progress by clicking the "Use xxxxx" link to allow the validation or disabling the android plugin, but I expect a few people to be confused by this along the way.

![image](https://user-images.githubusercontent.com/252189/118367010-7681d580-b555-11eb-8e3b-b4ea8cf140b7.png)

Reply:
Hello @lbradstreet and thanx for dropping by 😃 !

I am (was) aware of this because I am using both Android Studio and IntelliJ IDEA (although with Android plugin disabled) on a daily basis. 
However, I decided to include checksum validation, herewith rationale: 
- Gradle 7.0 git tag https://github.com/gradle/gradle/releases/tag/v7.0.0 notes mentions this: 
```
Upgrade Instructions

Switch your build to use Gradle 7.0 by updating your wrapper:

./gradlew wrapper --gradle-version=7.0 --gradle-distribution-sha256-sum=eb8b89184261025b0430f5b2233701ff1377f96da1ef5e278af6ae8bac5cc305
```
- issue is fixed and it will be released soon:
   - git commit: https://github.com/JetBrains/android/commit/1a79d8e81b5d5b021161a3456ec98406c0eed0ad
   - ticket https://youtrack.jetbrains.com/issue/IDEA-266355 mentions that solution is included into IntelliJ IDEA build **_211.7277_** 
- this is not an Android project after all


All-in-all: given a fact that current IntelliJ IDEA version is 2021.1.1:
```
IntelliJ IDEA 2021.1.1 (Ultimate Edition)
Build #IU-211.7142.45, built on April 30, 2021
```   
we can expect that next Intellij IDEA patch version 2021.1.2 will resolve this.

Reply:
Thanks @dejan2609! I think it’s a good change and we should leave it in. I was mostly commenting here in case someone else hits it before there’s a workaround.

Reply:
Intellij IDEA version 2021.1.2 has been released:
- https://blog.jetbrains.com/idea/2021/06/intellij-idea-2021-1-2
- https://youtrack.jetbrains.com/issue/IDEA-266355